prov/efa/src/rxr/rxr_pkt_entry.h

```diff
@@ -114,6 +114,17 @@ struct rxr_pkt_sendv {
  * is not registered (when it is unexpected or out-of-order). A new packet entry will be cloned
  * using endpoint's read_copy_pkt_pool, whose memory was registered.
  */
+
+struct efa_send_wr {
```

Comment:
As mentioned before, this should remain in `efa.h` because it used by both rdm and dgram

Reply:
The import hierarchy right now is that `efa.h` imports `rxr.h` and `rxr.h` does not import `efa.h`. So if we keep this in `efa.h`, we have to add `#import "efa.h"` to `rxr_pkt_entry.h` which leads to circular dependency issues and compilation failure

I can fix the import hierarchy in a later commit when we rename `efa` to `efa_dgram`, `rxr` to `efa_rdm` and move common code to common files.

Reply:
In that case, please add a todo comment here.

Reply:
done